build-system/tasks/process-github-issues.js

```diff
@@ -136,6 +136,13 @@ function updateGitHubIssues() {
           const issueLastUpdate = issue.updated_at;
           let biweeklyUpdate = true;
           let quartelyUpdate = true;
+          // if an issue is an pull request will skip it
```

Comment:
if an issue is **a** pull request, **we'll** skip it

Reply:
done